vocabularies/quality.ttl

```diff
@@ -0,0 +1,685 @@
+# baseURI: https://linked.data.qld.gov.au/vocab
+# imports: https://linked.data.qld.gov.au/def/dcat
+
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix dq-iso-19157: <https://linked.data.gov.au/def/ISO-19157/quality-dimension/> .
+@prefix dq-iso-25012: <https://linked.data.gov.au/def/ISO-IEC-25012/quality-dimension/> .
+@prefix dq-nsw: <https://linked.data.gov.au/def/NSW-OEH/quality-dimension/> .
+@prefix dqv: <http://www.w3.org/ns/dqv#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix sdo: <http://www.schema.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+dq-iso-19157:Dimensions
+  a skos:ConceptScheme ;
+  dcterms:source "ISO 19157"@en ;
+  rdfs:label "Dimensions of data quailty (ISO 19157)"@en ;
+  skos:prefLabel "Dimensions of data quailty (ISO 19157)"@en ;
+  dcterms:created "2020-11-16"^^xsd:date ;
+  dcterms:creator <http://linked.data.gov.au/def/gsq-alias/des> ;
+dcterms:modified "2020-11-16"^^xsd:date ;
+dcterms:publisher <http://linked.data.gov.au/def/gsq-alias/des> ;
+  skos:hasTopConcept dq-iso-19157:Completeness ;
+  skos:hasTopConcept dq-iso-19157:LogicalConsistency ;
+  skos:hasTopConcept dq-iso-19157:Metaquality ;
+  skos:hasTopConcept dq-iso-19157:PositionalAccuracy ;
+  skos:hasTopConcept dq-iso-19157:TemporalQuality ;
+  skos:hasTopConcept dq-iso-19157:ThematicAccuracy ;
+  skos:hasTopConcept dq-iso-19157:UsabilityElement .

```

Comment:
General comment: As all the declarations are reciprocal you can either list the top concepts in the concept scheme, or include a row in each concept of skos:topConceptOf <conceptScheme URL>. 

I personally find the latter a bit more manageable and readable for vocabularies with a large number of concepts, so it has become my default. But either way works, personal choice. You can do both too if you really want... but it's just extra lines for no additional information

Reply:
Will make note of that, probably start applying just keeping at the concept level. 
Was including in each based on a response from the SKOS testing tools.